core/kits/assets/js/globals/global-select-behavior.js

```diff
@@ -268,12 +268,34 @@ export default class GlobalControlSelect extends Marionette.Behavior {
 			onShow: () => {
 				// Put focus on the naming input.
 				this.ui.globalNameInput = this.confirmNewGlobalModal.getElements( 'widget' ).find( 'input' ).focus();
+				this.ui.confirmMessageText = this.confirmNewGlobalModal.getElements( 'widget' ).find( '.e-global__confirm-message-text' );
```

Comment:
Insert `this.confirmNewGlobalModal.getElements( 'widget' )` into a local variable